### PR TITLE
[FX-2279] Support multiple collections on fair pages

### PR DESCRIPTION
**Pull request by dzucconi**
Re: [FX-2245](https://artsyproduct.atlassian.net/browse/FX-2245)

* Displays up to 4 collections in on a **shelf**
* Switches articles display to left-to-right flowing grid

### CODE REVIEW

File: src/v2/Apps/Fair/Components/FairCollections/FairCollection.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+]
+
+interface FairCollectionProps {
+  collection: FairCollection_collection
+}
+
+export const FairCollection: React.FC<FairCollectionProps> = ({
+  collection,
+}) => {
+  const imageUrls = collection.artworks.edges.map(
```
Comment (sweir27) [inline src/v2/Apps/Fair/Components/FairCollections/FairCollection.tsx:24]: We don't have strict null-checking here, but you may want to make this more defensive.

File: src/v2/Apps/Fair/__tests__/FairApp.jest.tsx [hunk 2]
```diff
@@ -132,7 +132,7 @@ describe("FairApp", () => {
     expect(wrapper.find("FairHeader").length).toBe(1)
   })
 
-  it("renders articles if they are present", async () => {
+  fit("renders articles if they are present", async () => {
```
Comment (sweir27) [inline src/v2/Apps/Fair/__tests__/FairApp.jest.tsx:135]: `fit` intentional?

Comment (dzucconi) [reply src/v2/Apps/Fair/__tests__/FairApp.jest.tsx:135]: Nope! I keep doing this.

File: src/v2/Apps/Fair/Components/FairCollections/FairCollection.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Apps/Fair/Components/FairCollections/FairCollection.tsx]: Updated: ‧͙⁺˚*･༓☾ with more safety ☽༓･*˚⁺‧͙

**Pull request merged**